README.md

```diff
@@ -174,6 +176,17 @@ To be able to create your own data source implementation you need to implement t
 
 You can see a really simple data source implementation on our `mem` package.
 
+## Indexes
+
+`go-mysql-server` exposes a series of interfaces to allow you to implement your own indexes so you can speedup your queries.
+
+Taking a look at the main [index interface](https://github.com/src-d/go-mysql-server/blob/0d36af3c06c4c200a20e6e6f7eb9e29045e98b81/sql/index.go#L35), you must note a couple of constraints:
```

Comment:
inside the go-mysql-server we don't want to put the specific hash, each documentation link will be pointing to the correct place.

Reply:
ohh, what I meant is only possible on issues markdown:

https://github.com/src-d/go-mysql-server/blob/0d36af3c06c4c200a20e6e6f7eb9e29045e98b81/sql/index.go#L35-L52

Anyway, If it is possible, we should point to master or godoc instead to a specific commit hash.